packages/trader/src/App/Components/Elements/Modals/UrlUnavailableModal/url-unavailable-modal.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Button, Modal, StaticUrl, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+
+type TUrlUnavailableModalProps = React.PropsWithChildren<{
+    isMobile?: boolean;
+    isVisible: boolean;
+    onConfirm: () => void;
+}>;
+
+const UrlUnavailableModal = ({ isMobile, isVisible, onConfirm }: TUrlUnavailableModalProps) => (
+    <Modal
```

Comment:
I remember, that there was a problem, when we were showing 2 pop-ups at the same time (screenshot is pinned). It looked unstylish, especially when they (2 different modals) were sharing same `modal_root` (default for `<Modal />`). 
What I'm trying to say is that current modal also might face same problem 
<img width="1330" alt="Screenshot 2024-02-08 at 14 21 58" src="https://github.com/binary-com/deriv-app/assets/121025168/db3d76c1-caea-4818-bea6-7860ebdc2283">

Reply:
good catch, thank you for reminding me :)

Reply:
I've added a safeguard for this case in `trading.scss`. I couldn't reproduce this case myself but just in case. :) My modal will now appear only after all other modals in modal-root are closed, so that we don't affect any other flows and at the same time still inform a user that the URL entered is not available.